src/chart/tooltip.js

```diff
@@ -0,0 +1,87 @@
+import d3 from 'd3';
+import dataJoinUtil from '../util/dataJoin';
+import {noop} from '../util/fn';
+
+export default function() {
+
+    var labelWidth = 50,
+        decorate = noop;
+
+    var items = [
+        ['datum:', function(d) { return d.datum; }]
```

Comment:
The problem with that interface is I think it becomes quite unreadable in practice....

```
 tooltip()
    .labels(['high', 'low', 'open', 'close'])
    .values([
       function(d) { return d.high; },
       function(d) { return d.low; },
       function(d) { return d.high; },
       function(d) { return d.cose; },
    ])
```

It's hard to see the correlation between labels and values.

I also like the brevity of the current interface, it's similar to the way D3 expresses ranges